moduleroot/.travis.yml.erb

```diff
@@ -181,7 +182,7 @@ before_deploy:
 <% end -%>
 deploy:
   provider: puppetforge
-  user: <%= @configs['user'] %>
+  username: <%= @configs['user'] %>
```

Comment:
@rtib Have you validated that this still works with the Travis `puppetforge` deploy module? It looks like the field name changed between API versions and it's not clear that this provider is set to use the new version.

Reply:
Sure, I've released a maintenance update of my [cassandra](https://github.com/rtib/puppet-cassandra/tree/v2.1.2) module, which [Travis-CI](https://travis-ci.org/rtib/puppet-cassandra/builds/647276824) successfully deployed to [forge](https://forge.puppet.com/trepasi/cassandra).

Reply:
Prior to these changes, Travis was raising the above warnings.

Reply:
OK, sounds good!